extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -68,6 +68,16 @@ object PrivilegesBuilder {
     }
   }
 
+  private def isTempView(
+      tableIdent: TableIdentifier,
+      spark: SparkSession): Boolean = {
+    val parts = tableIdent match {
+      case TableIdentifier(table, None) => Seq(table)
+      case TableIdentifier(tblName, Some(dbName)) => Seq(dbName, tblName)
```

Comment:
it's better to use such code to have a better compatibility:
```scala
tableIdent.database match {
  case Some(db) =>
    Seq(db, tableIdent.table)
  case _ =>
    Seq(tableIdent.table)
}
```

Since the Spark master branch (3.4) will be:
`TableIdentifier(table: String, database: Option[String], catalog: Option[String])`